### PR TITLE
fix: Add role property to Box component

### PR DESCRIPTION
**Pull request by magnuh**

### CODE REVIEW

**Review by felicia-haggqvist — APPROVED** [review]

File: packages/box/index.js [hunk 1]
```diff
@@ -35,9 +36,13 @@ class WarpBox extends LitElement {
     });
   }
 
+  get _optOutRoleWithDefault() {
+    return this.role === '' ? nothing : this.role ?? 'region'
```
Comment (BalbinaK) [inline packages/box/index.js:40]: I'm not familiar with `nothing`, but if we want to make sure that `role` attribute is hidden when `''` is assigned to it, we could use `ifDefined(this._optOutRoleWithDefault)` directive in line 45 ([lit docs](https://lit.dev/docs/templates/directives/#ifdefined)).

Comment (magnuh) [reply packages/box/index.js:40]: https://lit.dev/docs/v1/lit-html/writing-templates/#rendering-nothing

Comment (BalbinaK) [reply packages/box/index.js:40]: Oh, that's cool! Thanks!

Comment (magnuh) [reply packages/box/index.js:40]: It gets a bit "messy" when we want a fallback AND still be able to remove the attribute. 😅

**Pull request merged**